### DISCUSSION TITLE
Add the ability to create checklists

### DISCUSSION THREAD

**Discussion by dvd-22** [ostoaleonardo/notes#2] [Feature requests]
As the title says. I'd like to have checklists. On Google Keep, you can create them, but then the whole note becomes a checklist. Having the ability to embed them within the contents of the note without turning it as a whole into a checklist would be awesome.

**Comment by ostoaleonardo** [comment]
I'm currently working on this feature. You'll be able to have a list per note, bulleted, numbered, or checklist.
